web/templates/stream_settings/browse_streams_list_item.hbs

```diff
@@ -14,6 +14,22 @@
                     </span>
                 </template>
 
+                <template id="cannot-unsubscribe-from-{{name}}-stream-tooltip-template">
+                    <span>
+                        {{#tr}}
+                            You are not allowed to unsubscribe from this channel.
```

Comment:
For some reason I am seeing empty tooltips for both streams user can and cannot unsubscribe from
[Screencast from 2025-11-07 18-55-07.webm](https://github.com/user-attachments/assets/216c8d14-9a16-474c-84e7-0267da77915d)